pubtools/exodus/_tasks/push.py

```diff
@@ -30,13 +32,21 @@ def run(self):
         publish_id = str(publish.get("id"))
         LOG.info("Publish ID: %s", publish_id)
 
-        for item in self.args.sync:
+        push_items = []
+        with Source.get(self.args.source) as source:
+            for push_item in source:
```

Comment:
Could you please make this tolerate (warn or ignore) unknown push item types? It's suggested in https://release-engineering.github.io/pushsource/userguide.html#processing-push-items to ensure the code doesn't break if pushsource library later adds support for new kinds of items.

Based on the way the code works later on, maybe rather than doing it by an isinstance check, the right way would be something like this:

```python
if push_item.src and len(push_item.dest) == 1:
    push_items.append(push_item)
else:
    LOG.warning("Skipping unsupported item: %s", push_item)
```

Basically if the item contains a src (file or directory) and a single dest, then we know how to handle it.  If not, then we don't.